packages/tooltip/src/components/tooltip/WithTooltip.md

```diff
@@ -0,0 +1,59 @@
+# WithTooltip
+
+`WithTooltip` can add a tooltip to any component.
+
+```
+<WithTooltip
+  placement={"bottom"}
+  content={<StandardText>Nice content on bottom</StandardText>}
+>
+  <StandardText>Hover me</StandardText>
+</WithTooltip>
+```
+
+This uses popper.js to position the tooltip.
+
+## `placement` prop
+
+Specifies which side of `children` the tooltip should be positioned.
+This is optional. Can be `"bottom"`, `"top"`, `"left"` or `"right"`.
+
+## `trigger` prop
+
+`trigger` can be set to `"hover"` or `"click"`.
+
+### Hover trigger
+
+When `hover`, the tooltip is displayed when mouse hovers over `children`.
+When the mouse no longer hovers, the tooltip is hidden again.
+
+### Click trigger
+
+When `click`, the tooltip is displayed when the user clicks on `children`.
+
+If `children` is a React node, it is automatically wrapped in `Clickable`.
+
+If you are using a component that has it's own click listener (`StandardButton` for example),
+you can use functional children to programmatically show the tooltip.
+
+See `render props` section below.
+
+Whenever using `click` trigger, `WithTooltip` will hide the tooltip when clicking
```

Comment:
Make it optional? It's better for performance (don't need clickOutside) and sometimes you want to force the user to make a decision in the tooltip.

Reply:
I'm not sure this is true, tooltip doesn't prevent the user from interacting with the rest of the webpage.

I agree for other popups, such as modal windows, etc, but not tooltip.

![Screenshot 2019-08-13 at 16 34 12](https://user-images.githubusercontent.com/1266041/63018963-6b8aa480-be9a-11e9-967d-40957f705c8d.png)